lib/core/config-schema.js

```
@@ -0,0 +1,204 @@
+        passwordPolicy: Joi.object({
+          minLength: Joi.number().integer().min(6).default(8),
+          requireUppercase: Joi.boolean().default(true),
+          requireLowercase: Joi.boolean().default(true),
+          requireNumbers: Joi.boolean().default(true),
+          requireSpecialChars: Joi.boolean().default(false)
+        }).default({})
+      }).default({}),
+      oauth: Joi.object({
+        google: Joi.object({
+          enabled: Joi.boolean().default(false),
+          clientId: Joi.string().when('enabled', { is: true, then: Joi.required() }),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)